### ISSUE TITLE
Weight rounding is off sometimes

### ISSUE THREAD

**Issue by JonnyBeeGod** [JonnyBeeGod/nSuns-5-3-1-iOS-issues#30]
![image_123986672](https://user-images.githubusercontent.com/2844335/36060008-5d570f72-0e40-11e8-947d-f3bf93ab93f8.JPG)

It should not be possible to get calculated weights like x.25 even with microloading set as weight. 

For example this week on my +1 squats I did 120kg x 5 as per the app at that time but now I notice it says that I done 118.75 for 5 which is throwing my figures off for next week. I find that adjusting my TM manually isn’t really fixing this as it’s giving me figures that I wouldn’t be able to do. I believe this is an issue with rounding is there a fix for this?

**Comment by pepablock** [comment]
I've been using the app for just about 2 weeks and on my second cycle I experienced this as well. I think the latest update that was pushed on Tuesday caused this issue, I remember checking what numbers I will be doing and after updating it messed up the numbers for the whole cycle. I Don't know how to fix it, but after I created a 3rd cycle weights seems to be rounded correctly again, so far only one cycle was affected for me.

Thanks for the awesome app, loving it!
